src/WebJobs.Script.WebHost/DependencyInjection/DependencyValidator/DependencyValidator.cs

```diff
@@ -54,7 +54,8 @@ private static ExpectedDependencyBuilder CreateExpectedDependencies()
                 .Optional<JobHostService>() // Missing when host is offline.
                 .Optional<FunctionsSyncService>() // Conditionally registered.
                 .OptionalExternal("Microsoft.AspNetCore.DataProtection.Internal.DataProtectionHostedService", "Microsoft.AspNetCore.DataProtection", "adb9793829ddae60") // Popularly-registered by DataProtection.
-                .OptionalExternal("Microsoft.Extensions.Diagnostics.HealthChecks.HealthCheckPublisherHostedService", "Microsoft.Extensions.Diagnostics.HealthChecks", "adb9793829ddae60"); // Popularly-registered by Health Check Monitor.
+                .OptionalExternal("Microsoft.Extensions.Diagnostics.HealthChecks.HealthCheckPublisherHostedService", "Microsoft.Extensions.Diagnostics.HealthChecks", "adb9793829ddae60") // Popularly-registered by Health Check Monitor.
+                .OptionalExternal("OpenTelemetry.Extensions.Hosting.Implementation.TelemetryHostedService", "OpenTelemetry.Extensions.Hosting", "adb9793829ddae60");
```

Comment:
The 3rd parameter is "assemblyPublicKeyToken" , which is wrong here. The correct one is **`"7bd6737fe5b67e3c"`** 
-- which you can get with "sn -T <assemblyName>" on VS developer command prompt, for example.

Please add a short comment here (something like enable OpenTelemetry.Net instrumentation library).

In addition, please add a row to the release_notes.md in the root of the repository like
- Enabled the use of OpenTelemetry.Net instrumentation library (#7253)

Reply:
Thank you for your review!